README.md

```diff
@@ -67,27 +69,31 @@ You can build your models upon `padertorch.Module` and `padertorch.Model`.
 Both expect a `forward` method which has the same functionality as the `forward` call of  `torch.nn.Module`: It takes data as input, applies some transformations, and returns the network output:
 
 ```python
-def forward(self, example):
-    x = example['x']
-    out = transform(x)
-    return out
+class MyModel(pt.Model):
+
+  def forward(self, example):
+      x = example['x']
+      out = transform(x)
+      return out
 ```
 
 Additionally, `padertorch.Model` expects a `review` method to be implemented which takes the data and output of the `forward` call as inputs from which it computes the training loss and metrics for logging in tensorboard:
```

Comment:
I would add something like "This is an example for a classification problem using the cross-entropy loss"